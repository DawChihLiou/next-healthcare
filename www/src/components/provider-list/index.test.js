import React from 'react';
import { render } from '@testing-library/react';

import Comp from './index';

describe('Provider List', () => {
  it('should match Snapshot', () => {
    const { asFragment } = render(<Comp />);
    expect(asFragment()).toMatchSnapshot();
  });
});
