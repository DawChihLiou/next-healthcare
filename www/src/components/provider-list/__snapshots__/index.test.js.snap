// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider List should match Snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg makeStyles-container-1"
  />
</DocumentFragment>
`;
